cloudbridge/cloud/providers/gce/services.py

```diff
@@ -273,11 +273,33 @@ class GCEImageService(BaseImageService):
 
     def __init__(self, provider):
         super(GCEImageService, self).__init__(provider)
+        self._public_images = None
+
+    PUBLIC_IMAGE_PROJECTS = ['centos-cloud', 'coreos-cloud', 'debian-cloud',
```

Comment:
If the PUBLIC_IMAGE_PROJECTS list is intended to be class private, then prefix this with an underscore.